wiki/Welcome/de.md

```diff
@@ -1,130 +1,119 @@
-Zuallererst möchten wir dich herzlichst empfangen und eine Umarmung für die Teilnahme an unserer Community geben. Lass uns erst ein paar Dinge durchgehen, bevor wir mit der Reise beginnen. ![osu!](Osu-boxart.png "fig:osu!")
+# Willkommen
 
-\_\_TOC\_\_
+![osu!](osu-boxart.png "osu!")
 
-Das Tutorial
-------------
+Zuallererst möchten wir dich herzlichst empfangen und eine Umarmung für die Teilnahme an unserer Community geben. Lass uns erst ein paar Dinge durchgehen, bevor wir mit der Reise beginnen. 
 
-Nachdem du das Spiel heruntergeladen und installiert hast, sollte beim ersten Starten sich das Spiel automatisch updaten. Im Startmenü siehst du einen großen osu! Knopf in der Mitte des Bildschirmes. Klicke zweimal darauf und wähle den "Solo" Modus. Bevor du irgendeine Beatmap in osu! zu spielen beginnst, ist es empfehlenswert, zuerst das ([Peter Lambert - osu! Tutorial](http://osu.ppy.sh/s/3756)) zu spielen. Du kannst auch mit Hilfe der Suchfunktion eine Beatmap in der Songauswahl finden, wir beschäftigen uns jedoch erst mit dem osu! Tutorial. Die Beatmap bringt dir das Wesentliche über die Hit-Circles, Slider und Spinner in osu! Standard bei. Um gewisse Spieltechniken - wie das Alternaten oder Streamen - zu erlernen, hilft nur spielen, spielen und noch mehr spielen. Verbaler Wissensaustausch hilft einem in diesem Punkt nicht weiter. Die dazugepackten Beatmaps neben den osu! Tutorial sollen auch als Lernhilfe dienen.
+## Das Tutorial
 
-**Achtung:** Wenn du aus einigen Gründen das osu! Tutorial nicht haben solltest, kannst du es [hier](http://osu.ppy.sh/b/22538) erneut herunterladen. Nach dem Tutorial bist du bereit eine Vielzahl von unterschiedlichen Beatmaps herunterzuladen!
+Nachdem du das Spiel heruntergeladen und installiert hast, sollte beim ersten Starten sich das Spiel automatisch updaten. Im Startmenü siehst du einen großen *osu!* Knopf in der Mitte des Bildschirmes. Klicke zweimal darauf und wähle den `Solo` Modus. Bevor du irgendeine Beatmap in *osu!* zu spielen beginnst, ist es empfehlenswert, zuerst das [Peter Lambert - osu! Tutorial](http://osu.ppy.sh/s/3756 "Peter Lambert - osu! Tutorial") zu spielen. Du kannst auch mit Hilfe der Suchfunktion eine Beatmap in der Songauswahl finden, wir beschäftigen uns jedoch erst mit dem *osu!* Tutorial. Die Beatmap bringt dir das Wesentliche über die *Hit-Circles*, *Slider* und *Spinner* in *osu!* Standard bei. Um gewisse Spieltechniken - wie das *Alternaten* oder *Streamen* - zu erlernen, hilft nur spielen, spielen und noch mehr spielen. Verbaler Wissensaustausch hilft einem in diesem Punkt nicht weiter. Die dazugepackten Beatmaps neben den *osu!* Tutorial sollen auch als Lernhilfe dienen.
```

Comment:
> sollte sich das Spiel beim ersten Start automatisch aktualisieren.

klingt besser

> "wir beschäftigen uns jedoch erst mit dem osu! tutorial". 

Statt `,` besser `-` 

> Verbaler Wissensaustausch 

klingt jetzt aber sehr hochgestochen

> "dazugepackt" 

is auch so ein blödes Wort. "mitgeliefert"?